cdbcli/main.py

```diff
@@ -56,7 +56,7 @@ def url(self):
 @click.argument('database', default='', required=False)
 def main(host, port, username, password, askpass, tls, database):
     if askpass:
-        password = getpass.getpass('Enter password: ')
+        password = prompt('Enter password: ')
```

Comment:
This echoes the password by default. There has to be a way in prompt-toolkit to mask out the password.